.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+services:
+  - docker
+
+dist: bionic
+
+language: ruby
+rvm:
+  - 2.5.5
+
+before_install:
+  - docker pull mongo:3.6
+  - docker pull redis
+  - docker run -d -p 127.0.0.1:27017:27017 mongo:3.6
+  - docker run -p 127.0.0.1:6379:6379 -d redis redis-server --appendonly yes
+  - export DB_PORT=127.0.0.1:27017 SKIP_RABBIT_MQ=true
+  - sudo apt install -y zlib1g-dev build-essential libssl-dev libreadline-dev libyaml-dev libsqlite3-dev sqlite3 libxml2-dev libxslt1-dev libcurl4-openssl-dev libffi-dev imagemagick libmagickwand-dev
+
+before_script:
+  - bundle exec rake assets:precompile
+
+script:
+  - bundle exec rspec spec
+
+after_success:
+  #Log in to the Docker CLI
+  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+
+  - docker build -t alejandro92/cenit:latest ./docker/cenit
```

Comment:
Hola @alainfd82 not should be nothing with `alejandro92` please move it to a cenit account

Reply:
I need access to Cenit account on Docker Hub.